src/cluster/managers/documentation/BBTagDocumentationManager.ts

```diff
@@ -0,0 +1,336 @@
+import { bbtag, limits, Subtag, SubtagSignature, SubtagSignatureValueParameter, SubtagType, tagTypeDetails, tagVariableScopeProviders } from '@blargbot/bbtag';
```

Comment:
General comment about this:
> isn't the information duplicated between the documentation manager and the website by the way?

Reply:
Yep, but only the copy. The documentation for the commands/subtags themselves are all generated from the same data on both the website and here